module/TaskManagement/src/TaskManagement/Service/NotifyMailListener.php

```diff
@@ -94,4 +95,31 @@ public function sendSharesAssignedInfoMail(Task $task, User $member)
 		$result = $this->mailService->send();
 		return $result->isValid();
 	}
+	
+	/**
+	 * Send email notification to all members with empty shares of $taskToNotify
+	 * @param Task $taskToNotify
+	 */
+	public function remindAssignmentOfSharesOnSingleTask(ReadModelTask $taskToNotify){
```

Comment:
"OfSingleTask" è lapalissiano: il parametro che prende in ingresso è uno ed un solo Task...
"ToNotify" è anch'esso una ripetizione: il metodo di chiama remind...
Quindi remindAssignmentOfShares(Task $task) è meglio :-)